pkgs/applications/networking/cluster/kops/default.nix

```diff
@@ -51,28 +51,8 @@ in rec {
 
   mkKops = generic;
 
-  kops_1_12 = mkKops {
-    version = "1.12.3";
-    sha256 = "0rpbaz54l5v1z7ab5kpxcb4jyakkl5ysgz1sxajqmw2d6dvf7xly";
-  };
-
-  kops_1_13 = mkKops {
-    version = "1.13.2";
-    sha256 = "0lkkg34vn020r62ga8vg5d3a8jwvq00xlv3p1s01nkz33f6salng";
-  };
-
-  kops_1_14 = mkKops {
-    version = "1.14.1";
-    sha256 = "0ikd8qwrjh8s1sc95g18sm0q6p33swz2m1rjd8zw34mb2w9jv76n";
-  };
-
-  kops_1_15 = mkKops {
-    version = "1.15.2";
-    sha256 = "1sjfd7pfi81ccq1dkgkh9xx6y94bqzlp727pvyf7l01x3d14z2b3";
-  };
-
   kops_1_16 = mkKops {
-    version = "1.16.0";
-    sha256 = "1b2lzf6b29rs5imbpqp8gnp3b511lk7jrm2f62y32gmx0gyjws6a";
+    version = "1.16.1";
+    sha256 = "08vy57ln0qar961sf9vbrh29f04qj5siqsfim1kqwvj5xrvwa39i";
   };
```

Comment:
We can afford to keep 2 versions I think. It makes updating scenarios a bit easier.

Reply:
So should I readd kops 1.15?

Reply:
Yeah if you don't mind. :+1 for the PR in either case.